### DISCUSSION TITLE
Using the CheckSpectrograms.py file

### DISCUSSION THREAD

**Discussion by KakoozaJerry** [coqui-ai/TTS#866] [General Q&A] [answered]
Hello , Does anyone know how to use this py file.

https://github.com/coqui-ai/TTS/blob/main/notebooks/dataset_analysis/CheckSpectrograms.ipynb

Where can I get the `config_transformer2.json` file.  I already have my dataset in the LJSpeech format.

Thanks

**Comment by erogol** [comment, marked as answer]
It is a notebook file. Use jupyter notebook to play with it.
